core/src/main/java/org/apache/accumulo/core/metadata/schema/MetadataSchema.java

```diff
@@ -167,6 +173,20 @@ public static Text getRow(TableId tableId, Text endRow) {
     public static class BulkFileColumnFamily {
       public static final String STR_NAME = "loaded";
       public static final Text NAME = new Text(STR_NAME);
+
+      public static long getBulkLoadTid(Value v) {
+        return getBulkLoadTid(v.toString());
+      }
+
+      public static long getBulkLoadTid(String vs) {
+        if (FateTxId.isFormatedTid(vs)) {
+          return FateTxId.fromString(vs);
+        } else {
+          // a new serialization format was introduce in 2.0. This code support deserializing the
+          // old format.
+          return Long.parseLong(vs);
```

Comment:
I think this might be causing a NumberFormat exception on empty string in the tests.

Reply:
Do you know which test?

Reply:
https://travis-ci.org/apache/accumulo/jobs/566489719 shows:

```
[ERROR]   TabletMetadataTest.testAllColumns:93 » NumberFormat For input string: ""
```